packages/teleport/src/Recordings/RecordingsList.tsx

```diff
@@ -112,7 +112,13 @@ const renderPlayCell = (
     );
   }
 
-  const url = cfg.getPlayerRoute({ clusterId, sid }, { recordingType });
+  const url = cfg.getPlayerRoute(
+    { clusterId, sid },
+    {
+      recordingType,
+      durationMs: recordingType === 'desktop' ? duration : undefined,
```

Comment:
when we want to explicitly mean nothing, i think it's better to use `null`

Reply:
I think `undefined` is more appropriate here, because `durationMs` is optional and my intention is to say "don't define this thing".